examples/cubes/cubes/src/main.rs

```diff
@@ -57,11 +57,11 @@ fn update_cubes_from_mods(
     mut commands: Commands,
     mut meshes: ResMut<Assets<Mesh>>,
     mut materials: ResMut<Assets<StandardMaterial>>,
-    mut events_out: EventReader<ModMessage>, // GET messages FROM mods
-    mut events_in: EventWriter<HostMessage>, // SEND messages TO mods
+    mut mod_messages: EventReader<ModMessage>,
+    mut host_messages: EventWriter<HostMessage>,
```

Comment:
We should keep the comments here. Given that this is an example, we want to be as clear as possible